requirements.txt

```diff
@@ -3,7 +3,7 @@ datasets
 numpy
 torch
 tokenizers>=0.13.3
-transformers==4.33.0
+transformers==4.41.0
```

Comment:
have all the models been tested? The other models do not care about the version of transformers except llama I think.